saleor/order/utils.py

```diff
@@ -1318,3 +1297,7 @@ def clean_order_line_quantities(order_lines, quantities_for_lines):
                     )
                 }
             )
+
+
+def order_qs_select_for_update():
```

Comment:
Can you move this closer to `order_lines_qs_select_for_update`? Having both closed, would be helpful in case, of searching for the function name, or understanding that, the usage of this helper should be mandatory or general code readability.

Reply:
Moved